components/ReserveForUserModal.stories.ts

```diff
@@ -0,0 +1,42 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import ReserveForUserModal from '@/components/ReserveForUserModal';
+import { userEvent, within, waitFor, fn } from '@storybook/test';
+import { expect } from '@storybook/jest';
+
+const meta: Meta<typeof ReserveForUserModal> = {
+  title: 'Donations/ReserveForUserModal',
+  component: ReserveForUserModal,
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const ReserveForUserModalWithNoEnquiries: Story = {
+  args: {
+    itemId: 2,
+    name: 'Mark as Reserved',
+    requestedToReserveUserIds: [],
+    onReserveStatusChange: fn(),
+  },
```

Comment:
we need to also test that when there have been no enquiries we still get the behaviour we want:

```ts
  play: async ({ canvasElement, args }) => {
    const canvas = within(canvasElement);
    const reserveButton = canvas.getByText(args.name);
    await userEvent.click(reserveButton);
    await expect(
      canvas.getByText('Nobody has asked about this item yet.')
    ).toBeVisible();
  },

```